src/ex4/server/clients/PokemonClient.js

```diff
@@ -0,0 +1,52 @@
+
+import  {promises as fs } from "fs";
+import axios from 'axios';
+export default class PokemonClinet {
+  constructor() {
+  }
+  fetchPokemon(arr) {
+    const responses = [];
+    arr.forEach((id) => {
+      const data = axios.get("https://pokeapi.co/api/v2/pokemon/" + id);
+      responses.push(data);
+    });
+    return Promise.all(responses).then((res) => (res.map(r => r.data)));
+  }
+
+  async checkByPokemonName(name) {
+    let pokemonsArrList =await  this.readFromPokemonDataBaseFile();
+    if(pokemonsArrList===undefined){
+    const response = await axios.get(`https://pokeapi.co/api/v2/pokemon?limit=100000&offset=0`);
```

Comment:
```
params = {limit: 100000, offset: 0 }
await axios.get('https://pokeapi.co/api/v2/pokemon', { params });
```

Im pretty sure axios allows to send params and it wrap it by itself. please take a look if possible